hardhat.config.ts

```diff
@@ -52,6 +54,26 @@ const config: HardhatUserConfig = {
       chainId: 30,
       accounts: getAccounts("mainnet"),
     },
+    hardhat: {
+      ...(forkEnabled
+        ? {
+            forking: {
+              url:
+                process.env.FORK_NETWORK_URL ??
```

Comment:
If we read the env only once is easier to manage later in cases where we add/remove/modify vars, can you move this one and the below one there to the begging of the file? otherwise we'll start having process.env across the whole project